sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -407,16 +407,13 @@ export class ManagementClient extends LinkEntity {
           const initOperationStartTime = Date.now();
 
           try {
-            await waitForTimeoutOrAbortOrResolve({
```

Comment:
Nit: Now that we are not using `waitForTimeoutOrAbortOrResolve`, you can delete the file `timeoutAbortSignalUtils.ts` file. Or do that in a separate PR